docs/codespace_pr_reviews.md

```diff
@@ -0,0 +1,29 @@
+# Using Codesapces to review PRs without pulling a repo to your machine
+
```

Comment:
I'm assuming that the correct versions of packages etc will run within codespaces, so it's not necessary to have those updated locally. If that's correct, perhaps a short blurb explaining that would be helpful here.

Reply:
I added that context in, explaining that is *why* you have to wait a while when you first start the codespace.